recycling-server/db/migrate/20170302234459_create_categories.rb

```diff
@@ -4,6 +4,12 @@ def change
       t.string :name, null: false
       t.text :description
       t.string :image_link
+      t.string :recycle_class # Five digit string, "abcde"
```

Comment:
it's like summarizing many-to-many relationship into a single attribute. btw, the client has replied that each item can only belong to one recycle class